python/deepgram-language-detection/main.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import json
+
+
+def errorMessage(res, message):
+  return res.json({
+      "success": False,
+      "message": message,
+  })
+
+
+def main(req,res):
+  try:
+    #fileUrl and DEEPGRAM_API_KEY is required
+    file_url = req.payload['fileUrl']
+    api_key = req.variables.get('DEEPGRAM_API_KEY')
+
+  except Exception:
+    return errorMessage(res, 'Payload must contain fileUrl and DEEPGRAM_API_KEY.')
```

Comment:
```suggestion
    return errorMessage(res, 'Request must contain the File URL in the payload and the Deepgram API Key as a variable.')
```

The API Key is technically not a part of the request payload, but a part of the variables, so I recommend the change.